src/components/builder/is-sensitive-data.tsx

```diff
@@ -19,6 +19,7 @@ const IsSensitiveData = () => {
         />
       }
       tooltip={tooltip}
+      disabled={disabled}
```

Comment:
I suspect you mean to use `readonly` here rather than `disabled`. With the Formik state, there's no difference, but in pure HTML semantics a `disabled` form input is **NOT** submitted to the server, while a readonly input is.

The value of the property/field is important here for us, so I'd rather go for readonly than disabled.

Formio itself also famously gets this wrong where they use the `disabled` property when they actually mean `readonly`.

Reply:
Nice, I didn't know about this important difference.